app/javascript/App.jsx

```diff
@@ -2,11 +2,14 @@ import React from 'react';
 import { Container } from 'react-bootstrap';
 import { Outlet } from 'react-router-dom';
 import Header from './components/shared/Header';
+import { useAuth } from './contexts/auth/AuthProvider';
 
 export default function App() {
+  const currentUser = useAuth();
+
   return (
     <>
-      <Header />
+      {currentUser?.signed_in && <Header /> }
```

Comment:
That will introduce the fallowing bug:
>Users won't be able to login since they aren't authenticated, they won't see the HEADER

Reply:
True

Reply:
Yeah we'll need to them to the index for now. I'll add it to this pr

Reply:
Added a new homepage component with the 2 buttons. Once we get the mockup for the home page, we can make it look proper